swagger-spec/api/index.html

```diff
@@ -1,7 +1,7 @@
 {
   "swaggerVersion": "1.2",
   "apiVersion": "",
-  "basePath": "https://127.0.0.1:6443",
+  "basePath": "https://10.10.10.10:6443",
```

Comment:
Swagger-ui was using it, but we made that read-only so it is not using this any longer.
I dont know of any other usage of this field.